tests/fixtures/unicode-set.js

```diff
@@ -173,13 +173,15 @@ const unicodeSetFixtures = [
 	{
 		pattern: '[\\q{sA}asb]',
 		flags: 'iv',
-		expected: '(?:sA|[abs])'
+		expected: '(?:sa|[abs])',
+		expectedFlags: 'iu'
 	},
 	{
 		pattern: '[\\q{sA}asb]',
 		flags: 'iv',
 		options: TRANSFORM_U,
-		expected: '(?:[s\\u017F]A|[abs\\u017F])'
+		expected: '(?:[s\\u017F]a|[abs\\u017F])',
```

Comment:
Implement the `scf()` approach. For any class set operations, if the `i` flag is enabled or the `i` modifier is kept, the output will be always composed of "lowercase" characters, specifically the result set of the `scf()`.

Reply:
This case has been improved by skipping SCF on the most common union set operation.